wacruit/src/settings.py

```
@@ -9,6 +9,7 @@
 class Settings(BaseSettings):
     env: Literal["dev", "prod", "local", "test"] = "local"
     sql_echo: bool = False
+    TOKEN_SECRET: str = "secret"
```

Ruff (0.14.3)

Possible hardcoded password assigned to: "TOKEN_SECRET"

(S105)